iroh-ctl/Cargo.toml

```diff
@@ -26,3 +27,4 @@ libp2p = { version = "0.48", default-features = false }
 cid = "0.8.5"
 multiaddr = "0.14.0"
 bytes = "1.1.0"
+async-stream = "0.3.3"
```

Comment:
Don't we mostly use `futures` streams?

Reply:
It is a crate to help implement streams, not a different kind of stream